src/read/elf/relocation.rs

```diff
@@ -31,7 +31,7 @@ impl RelocationSections {
         let mut relocations = vec![0; sections.len()];
         for (index, section) in sections.iter().enumerate().rev() {
             let sh_type = section.sh_type(endian);
-            if sh_type == elf::SHT_REL || sh_type == elf::SHT_RELA {
+            if sh_type == elf::SHT_REL || sh_type == elf::SHT_RELA || sh_type == elf::SHT_CREL {
```

Comment:
I don't think this achieves anything without the corresponding support in `ElfRelaIterator`.